src/extensions/NEventExtension.ts

```diff
@@ -38,10 +32,11 @@ export const NEventExtension = Node.create({
 
```

Comment:
the extension options needs to be declared with it's defaults, otherwise it will be empty.

```ts
  addOptions() {
    return {
      getRelayHints() {
        return []
      },
    }
  },

```